Add RPNeko65 Keyboard (#21747)
qmk/qmk_firmware: keyboards/lendunistus/rpneko65/rev1/readme.md

Reviewer: :+1: 

![image](https://github.com/qmk/qmk_firmware/assets/18669334/414689ae-d334-449c-ac6c-8b3234672276)

# rpneko65

> ![rpneko65](https://raw.githubusercontent.com/lendunistus/rpneko65/rev1/images/rev1-pcb-purple.png)

Reviewer 2: Please could this image be re-hosted on imgur

> 

Reviewer 2: ```suggestion
*PCB rear side*

```

A multi-layout PCB for the Bakeneko65

* Keyboard Maintainer: [lendunistus](https://github.com/lendunistus)
* Hardware Supported: [Bakeneko65 V3](https://github.com/kkatano/bakeneko-65)
* Hardware Availability: [Open source on GitHub](https://github.com/lendunistus/rpneko65)

Make example for this keyboard (after setting up your build environment):

>     make lendunistus/rpneko65:default
> 
> Flashing example for this keyboard:
> 
>     make lendunistus/rpneko65:default:flash

Reviewer 2: ```suggestion
    make lendunistus/rpneko65/rev1:default

Flashing example for this keyboard:

    make lendunistus/rpneko65/rev1:default:flash
```

# rpneko65

> ![rpneko65](https://i.imgur.com/QUIUSqJ.png)

Reviewer 3: ```suggestion
![rpneko65](https://i.imgur.com/QUIUSqJh.png)

```

This renders a lower resolution version that is a bit more friendly to github markdown.

# rpneko65

> ![rpneko65](https://i.imgur.com/QUIUSqJh.png)
> 
> *PCB rear side*

Reviewer: If I remember correctly, this should put a single line break after the image, with the text immediately below like a caption. But I can't check because my Markdown editor has stopped working. :slightly_frowning_face: 

```suggestion
![rpneko65](https://i.imgur.com/QUIUSqJh.png)\
*PCB rear side*
```

A multi-layout PCB for the Bakeneko65

* Keyboard Maintainer: [lendunistus](https://github.com/lendunistus)

> * Hardware Supported: [Bakeneko65 V3](https://github.com/kkatano/bakeneko-65)

Author: yes, if the description wasn't clear enough

> A multi-layout PCB for the Bakeneko65
> 
> * Keyboard Maintainer: [lendunistus](https://github.com/lendunistus)
> * Hardware Supported: [Bakeneko65 V3](https://github.com/kkatano/bakeneko-65)

Reviewer: Edited for clarity.

```suggestion
A multi-layout PCB designed to fit the Bakeneko65 case by kkatano.

* Keyboard Maintainer: [lendunistus](https://github.com/lendunistus)
* Hardware Supported: RPNeko65 PCB; [Bakeneko65 V3 Case](https://github.com/kkatano/bakeneko-65)
```
